docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
+
+9. **Integrated load-testing** <sup>*</sup> Configure and run load tests against your microservices using [Codewind's performance dashboard](https://www.youtube.com/watch?v=nfJt3f5TUvc). Use the graphs to compare your repeated load test results and iterate on performance improvements. 
+
+<sup>*</sup> where supported.
+
+## What Integrated Development Environments (IDEs) does Codewind support?
+
+* Codewind is available as an extension for [**VS Code**](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) and [**Eclipse**](https://marketplace.eclipse.org/content/codewind) for local development work. In this setup, your integrated development environment is located locally, but the build and run itself can be done either locally or remotely.
+
+* If instead you prefer to develop your code from within the cloud enviroment where your microservice will be built and run, you can use Codewind for [**Eclipse Che**](https://www.eclipse.org/codewind/mdt-che-installinfo.html).
+
+## What cloud-native technology does Codewind support?
+When it comes to running your containerised application on the cloud, Codewind supports [Kubernetes](https://kubernetes.io/) and [Openshift](https://www.openshift.com/) for container orchestration. 
+
+## How does Codewind support my development architecture?
+Codewind supports several architectures, so you can use your local, test and deployment infrastructure however suits your teams. 
+
+* **Support for hosting your development environment locally or remotely** Your code development, which takes place within your IDE, can be located either locally, or on your cloud. 
+
+* **Support for building and running your containers locally or remotely** The building and running of your microservice is actioned by the Codewind Server, which can be located locally or on your cloud.
+
+This means that Codewind can be used in 3 different setups:
+
+1. **local configuration** - your IDE is hosted on your desktop. The building and running of your microservice also takes place on your desktop, via your local Docker Desktop installation.
+
+2. **fully hosted configuration** - both your IDE and the Codewind server are hosted on your cloud. In this setup, your code development takes place in the same location that your microservices are built and run.
+
+3. **remote configuration** - your IDE is hosted on your desktop, but the building and running of your microservice is undertaken on your cloud environment by the Codewind server. Using secure HTTPS connections, you connect to your multiple cloud-hosted microservices from the Codewind panel in your locally-hosted IDE.
+
+
+
+## What pre-configured containerised projects does Codewind provide?
+Codewind provides pre-configured project templates ADD URL covering several languages including Node.js, Java, Python and Swift, and several frameworks such as Express, Spring Boot and Open Liberty, with more on the way.
```

Comment:
- It would be good to add the url before merging to master. If we don't think we'll get the URL for a while, maybe write a note somewhere else so we can remind ourselves to add it and then remove the "ADD URL" text from the PR? I don't want "ADD URL" to accidentally get included in the website. 😅

Reply:
- "preconfigured" doesn't need a hyphen.
- comma after "several languages."
- comma after "several frameworks."
- comma after "Spring Boot"
- Remove "with more on the way."

Reply:
Good spot on the 'ADD URL' typo!

Reply:
done